hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -70,6 +71,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>bcprov-jdk15on</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-hdds-docs</artifactId>
```

Comment:
Repated again, it is already defined in L44

Reply:
Good catch, thanks. I removed the deplication (but added `provided` scope to the existing entry)